### PR TITLE
Issue #DP-296 feat: Filters on program table

### PR DESCRIPTION
**Pull request by ahghatol**

### CODE REVIEW

File: src/service/programService.js [hunk 1]
```diff
@@ -1016,6 +1018,50 @@ function programList(req, response) {
...
+    else if (key === 'content_types' && value) {
+      res[Op.or] = _.map(data.request.filters[key], (val) => {
+        return Sequelize.literal(`'${val}' = ANY (content_types)`);
+      });
+      delete data.request.filters[key];
+      return {
+        $and : res
+      }
+    }
+    else if ((key === 'nomination' || key === 'contribution') && value) {
```
Comment (twsvaishali) [inline src/service/programService.js:1045]: have nomination_enddate and content_submission_enddate as the filters

File: src/service/programService.js [hunk 2]
```diff
@@ -1077,20 +1123,23 @@ function programList(req, response) {
       });
   } else if (data.request.filters && data.request.filters.role && data.request.filters.user_id) {
     const promises = [];
-    let status = data.request.filters.status && _.map(data.request.filters.status, x => "'" + x + "'" ) || '';
```
Comment (twsvaishali) [inline src/service/programService.js:1080]: how this status filter is handled now?

Comment (twsvaishali) [reply src/service/programService.js:1080]: @ahghatol

File: src/service/programService.js (hunk 1 above)
Comment (ahghatol) [reply src/service/programService.js]: Done

File: src/service/programService.js (hunk 2 above)
Comment (ahghatol) [reply src/service/programService.js:1080]: As discussed done

**Pull request merged**